insights/parsers/tests/test_snmp.py

```diff
@@ -50,3 +152,15 @@ def test_snmp():
     stats = TcpIpStats(context_wrap(PROC_SNMP_NO))
     snmp_stats = stats.get("Ip")
     assert snmp_stats is None
+
+    stats = TcpIpStatsIPV6(context_wrap(PROC_SNMP6))
```

Comment:
Better separate the test for TcpIpStatsIPV6 to another test function e.g. `test_snmp6`?